tests/test_compute.py

```diff
@@ -0,0 +1,64 @@
+import os
+import numpy
+import pytest
+from RIAssigner.compute import CubicSpline, Kovats
+
+from tests.fixtures import indexed_data, non_indexed_data, reference_alkanes
+from tests.fixtures.data import load_test_file
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+data_location = os.path.join(here, "data")
+
+@pytest.mark.parametrize('this, other, expected', [
+    [CubicSpline(), CubicSpline(), True],
+    [Kovats(), Kovats(), True],
+    [CubicSpline(), Kovats(), False],
+    [Kovats(), object(), False],
+    [CubicSpline(), object(), False],
+])
+def test_equal(this, other, expected):
+    actual = (this == other)
+    assert actual == expected
+
+
+@pytest.mark.parametrize('method', [Kovats, CubicSpline])
+def test_construct(method):
+    compute = method()
+    assert compute is not None

```

Comment:
Consider marking this test as an dependency for all the other tests in this file using [pytest-dependency](https://pytest-dependency.readthedocs.io/en/stable/usage.html).

Reply:
I think this is a bit of an overkill since the constructor should actually never fail - I think the dependencies are more suitable for systems where a failure is somewhat of an expected event